<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tessera</string>
  </property>
  <widget class="QLabel" name="labelCodice">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Codice</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Iscrizione</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPunti">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Punti</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modifica">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifica</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_elimina">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimina</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
